<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background-color:black;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addLightButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Add Light</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectColourButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Colour Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colourPickerButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Choose Colour</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSequenceButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Save Sequence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Open Sequence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoSaveSequenceButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Auto-save sequence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePlaybackButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Save Playback</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPlaybackButton">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Next Playback</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousPlaybackButton">
    <property name="geometry">
     <rect>
      <x>1780</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Previous Playback</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colourReplaceButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Replace Colour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editSequenceButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Edit Sequence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addWholeRigButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Add whole rig</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPlaybackLabel">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
